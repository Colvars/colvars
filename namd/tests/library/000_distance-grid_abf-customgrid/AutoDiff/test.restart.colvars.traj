# step         one                   fa_one                
          20    3.19504109859627e+00  0.00000000000000e+00 
          21    3.19102191972704e+00  0.00000000000000e+00 
          22    3.18709720675364e+00  0.00000000000000e+00 
          23    3.18337516035850e+00  0.00000000000000e+00 
          24    3.17995068524441e+00  0.00000000000000e+00 
          25    3.17690060023898e+00 -1.70733191351305e+00 
          26    3.17425438901656e+00 -4.49848369146902e+00 
          27    3.17200310573942e+00 -8.13196505762149e+00 
          28    3.17010441301305e+00 -1.23579104029288e+01 
          29    3.16848920515434e+00 -1.69399919425882e+01 
          30    3.16706897911795e+00 -1.80599464168527e+01 
          31    3.16579864457612e+00 -1.88459693108695e+01 
          32    3.16463211540264e+00 -1.93446440423366e+01 
          33    3.16352515122836e+00 -1.96053998551592e+01 
          34    3.16243794428299e+00 -1.96786766893698e+01 
          35    3.16133719773243e+00 -1.96136510459479e+01 
          36    3.16019748810951e+00 -1.94558499141651e+01 
          37    3.15900181308568e+00 -1.92450798507875e+01 
          38    3.15774152752143e+00 -1.90145989083226e+01 
          39    3.15641592564363e+00 -1.87913372766038e+01 
          40    3.15503164800165e+00 -1.85966225976150e+01 
