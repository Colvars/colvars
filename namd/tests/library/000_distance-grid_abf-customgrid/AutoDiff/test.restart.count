# 1
#  1.00000000000000e+00  2.00000000000000e-01         20  0

  1.10000000000000e+00                      0
  1.30000000000000e+00                      0
  1.50000000000000e+00                      0
  1.70000000000000e+00                      0
  1.90000000000000e+00                      0
  2.10000000000000e+00                      0
  2.30000000000000e+00                      0
  2.50000000000000e+00                      0
  2.70000000000000e+00                      0
  2.90000000000000e+00                      0
  3.10000000000000e+00                     21
  3.30000000000000e+00                     19
  3.50000000000000e+00                      0
  3.70000000000000e+00                      0
  3.90000000000000e+00                      0
  4.10000000000000e+00                      0
  4.30000000000000e+00                      0
  4.50000000000000e+00                      0
  4.70000000000000e+00                      0
  4.90000000000000e+00                      0
