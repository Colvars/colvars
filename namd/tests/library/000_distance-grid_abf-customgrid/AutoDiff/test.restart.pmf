# 1
# -2.50000000000000e-01  5.00000000000000e-01         21  0

  0.00000000000000e+00   1.05880709883240e+01
  5.00000000000000e-01   1.05880709883240e+01
  1.00000000000000e+00   1.05880709883240e+01
  1.50000000000000e+00   1.05880709883240e+01
  2.00000000000000e+00   1.05880709883240e+01
  2.50000000000000e+00   1.05880709883240e+01
  3.00000000000000e+00   1.05880709883240e+01
  3.50000000000000e+00   1.05880709883240e+01
  4.00000000000000e+00   1.05880709883240e+01
  4.50000000000000e+00   1.05880709883240e+01
  5.00000000000000e+00   1.05880709883240e+01
  5.50000000000000e+00   1.28975968951650e+00
  6.00000000000000e+00   0.00000000000000e+00
  6.50000000000000e+00   0.00000000000000e+00
  7.00000000000000e+00   0.00000000000000e+00
  7.50000000000000e+00   0.00000000000000e+00
  8.00000000000000e+00   0.00000000000000e+00
  8.50000000000000e+00   0.00000000000000e+00
  9.00000000000000e+00   0.00000000000000e+00
  9.50000000000000e+00   0.00000000000000e+00
  1.00000000000000e+01   0.00000000000000e+00
