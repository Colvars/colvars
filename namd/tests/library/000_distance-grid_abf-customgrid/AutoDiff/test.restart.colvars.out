colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: Summary of compile-time features available in this build:
colvars:   - SMP parallelism: enabled (num. threads = 4)
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: available
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: The final output state file will be "test.restart.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "one"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 0.5
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 10
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { one }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 10
colvars:   # minSamples = 5 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # lowerBoundaries = { 1 }
colvars:   # upperBoundaries = { 5 }
colvars:   # widths = { 0.2 }
colvars:   # integrate = on [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Optimal rotation via flexible fitting:
colvars: - distance colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - ABF colvar bias implementation:
colvars: - Internal-forces free energy estimator:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - NAMD engine:
colvars: - Scalable center-of-mass computation (NAMD):
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: ----------------------------------------------------------------------
colvars: Loading state from text file "test.colvars.state".
colvars:   Restarting collective variable "one" from value: 3.19504
colvars:   Restarted abf bias "abf1" with step number 20.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (2 scalable groups, 8 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Current simulation parameters: initial step = 20, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "one":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "one":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: The restart output state file will be "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.restart.colvars.traj".
colvars: Saving collective variables state to "test.restart.tmp.colvars.state".
colvars: Saving collective variables state to "test.restart.colvars.state".
